tests/test_wheelfile.py

```diff
@@ -172,3 +173,19 @@ def test_attributes(tmpdir_factory, wheel_path):
         info = zf.getinfo('test-1.0.dist-info/RECORD')
         permissions = (info.external_attr >> 16) & 0o777
         assert permissions == 0o664
+
+
+def test_directories(tmpdir, wheel_path):
+    build_dir = tmpdir
+    sub_dir = build_dir / 'sub'
+    sub_dir.mkdir()
+    (sub_dir / '__init__.py').write_text('', encoding='utf-8')
```

Comment:
Why create a file in the directory? Wasn't the point to ensure that *empty* directories get included, or did I misunderstand?

Reply:
Good question. I decided that it's probably better to match the behavior observed in other tools that generate zip files from a tree and create directory entries for every directory. But I do take from your point that the test should probably also ensure that directory entries are created for empty directories.

Reply:
Updated in 0e94b10